style.css

```diff
@@ -0,0 +1,59 @@
+#main-doc {
+    width: 100%;
+    height: 100%;
+}
+
+#navbar {
+    position:fixed;
+    width: 300px;
+    height: 100%;
+    background-color: aqua;
+}
+#navbar a {
+    display: block;
+    text-decoration: none;
+    padding: 20px;
+    color:black;
+    border: 1px solid #999
+}
+#navbar header {
+    font-size:2em;
+    padding:10px;
+    text-align: center;
+}
+
+#info {
+    margin-left: 350px;
+}
+.main-section{
+    margin-left:300px;
+    padding-left:10px;
+}
+.main-section header{
+    font-size:2em;
+
+}
+body{
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+
+@media only screen and (max-width: 850px) {
+    
+    #navbar {
+    
+        height: auto;
+        position: relative;
+    
+    }
+    .main-section{
+        margin-left:0px
+    }
+      }
+
+code {
```

Comment:
Nice work using media queries, but I recommend you to check the defined max-width because if you notice the layout of your website changes too soon leaving some extra space on the right side of the navbar. 
![image](https://user-images.githubusercontent.com/95243996/159191337-697da3c1-a3c8-473f-9ae3-a90aba6919d8.png)